products/clamav.md

```diff
@@ -7,6 +7,18 @@ releasePolicyLink: https://docs.clamav.net/faq/faq-eol.html
 changelogTemplate: "https://github.com/Cisco-Talos/clamav/releases/tag/clamav-__LATEST__"
 releaseDateColumn: true
 
+identifiers:
+-   repology: clamav
+-   cpe: cpe:/a:cisco:clamav
+-   cpe: cpe:2.3:a:cisco:clamav
```

Comment:
i added this one because of the 'changelogTemplate' Link which leads to the clamav repo created by the organizational account of the talos group at cisco. so i assume the CPE was correct. i haven't been able to create a CPE checker script for all files (because it was buggy). Do you check the CPEs manually for each product?